### PR TITLE
Feature/friendly url

### LINKED ISSUE #2279: Understandable URLs for APIs and organizations
It might not sound a big issue but we need to have human understandable URLs for APIs.
This hardly is one: https://apinf.io/api/TYf33t5B6D6fwamKt (APInf Catalog REST API). Suggestion is to make it something like https://apinf.io/apis/APInf-Catalog-REST-API 

Note that in path I used plural "apis" which is the practice in REST APIs (in our catalogue API as well). This would make REST API and web behave with the same logic here. This would be just additional fine tuning and part of productisement.

### PR DESCRIPTION
**Pull request by marla-singer**
Closes #2279

### CODE REVIEW

File: apis/collection/server/publications.js [hunk 1]
```diff
@@ -8,8 +8,12 @@ import { Meteor } from 'meteor/meteor';
 import { check } from 'meteor/check';
 
 // Collection imports
```
Comment (jawidahmadi) [inline apis/collection/server/publications.js]: line 9 empty space may be removed.

File: feedback/client/form/autoform.js [hunk 2]
```diff
@@ -10,11 +10,18 @@ import { Modal } from 'meteor/peppelg:bootstrap-3-modal';
 import { TAPi18n } from 'meteor/tap:i18n';
 import { sAlert } from 'meteor/juliancwirko:s-alert';
```
Comment (jawidahmadi) [inline feedback/client/form/autoform.js]: line 12 no need empty space.

**Comment by brylie** [comment]
@jawidahmadi did you test this or read the code? There are changes in almost 20 files, and it would be surprising that no feedback was necessary.

File: backlog/client/form/autoform.js [hunk 3]
```diff
@@ -8,12 +8,18 @@ import { AutoForm } from 'meteor/aldeed:autoform';
...
+import Apis from '/apis/collection';
+
 AutoForm.hooks({
   apiBacklogItemForm: {
     before: {
       insert (backlogItem) {
-        // Attach API Backend Id to backlog item
-        backlogItem.apiBackendId = FlowRouter.getParam('_id');
+        // Get related API document
+        const api = Apis.findOne({ slug: FlowRouter.getParam('slug') });
```
Comment (brylie) [inline backlog/client/form/autoform.js]: A more conventional way to do this in our code would be to pass the `apiId` in to the autoform template as an argument.

File: apis/collection/server/publications.js [hunk 4]
```diff
@@ -70,15 +74,54 @@ Meteor.publish('latestPublicApis', (limit) => {
...
+          return Meteor.users.find(
+            { _id: { $in: api.authorizedUserIds } },
+            { fields: { username: 1, emails: 1, _id: 1 } }
+          );
+        },
+      },
+      {
+        find (api) {
+          // Make sure a user can edit this API
+          if (this.userId) {
```
Comment (brylie) [inline apis/collection/server/publications.js]: This only seems to check that a user is logged in, while the comment indicates we want to 'Make sure a user can edit this API'. Does this code do what the comment indicates?

Comment (marla-singer) [inline apis/collection/server/publications.js]: @brylie I can make the separate PR with improvements

**Pull request merged**